code/frontend/cypress/e2e/admin.spec.cy.ts

```diff
@@ -0,0 +1,13 @@
+Cypress.config('baseUrl', Cypress.env('BASE_URL_ADMIN_WEBSITE'))
+
+describe('the cwyd admin website', () => {
+  before(() => {
+    const myVariable = Cypress.env('BASE_URL_ADMIN_WEBSITE');
+    cy.log('My variable:', myVariable);
+    cy.visit('/Ingest_Data');
+  });
+
+  it('allows file upload', () => {
+    cy.get('input[type=file]', { timeout: 30000 }).selectFile('cypress/fixtures/ingest.txt', { force: true });
```

Comment:
Should we add a verification that the processing/upload successful message is then displayed?

Reply:
Done